### PR TITLE
Simple features

### PR DESCRIPTION
**Pull request by szok**
- Change Back Url in record request detail
- Add message after add or save record
- Focus to name or content depending on the type of record
- Improve autocomplete
- Add flash message

zrzut ekranu 2016-07-14 o 12 02 24
zrzut ekranu 2016-07-14 o 12 16 43
zrzut ekranu 2016-07-14 o 12 16 52

### CODE REVIEW

**Comment by dominikkowalski** [comment]
:ok:

File: ui/static/app/record/record-detail.component.ts [hunk 1]
```diff
@@ -190,16 +207,24 @@ export class RecordDetailComponent implements OnInit {
 
     onChangeType() {
       if (this.isCreate) {
-        if (this.record.type === "A" || this.record.type === "MX") {
```
Comment (mkurek) [inline ui/static/app/record/record-detail.component.ts:193]: what's the difference between type A and MX here? I don't see any.

Comment (szok) [inline ui/static/app/record/record-detail.component.ts:193]: if A is select focus set to content field
if MX is select focus set to name field

**Comment by krzysztofzygmunt** [comment]
:ok:

**Pull request merged**